packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -1270,3 +1266,26 @@ export default decorate([
     )
   },
 ])
+
+export default decorate([
+  addSubscriptions({
+    remotes: subscribeRemotes,
+  }),
+  provideState({
+    initialState: () => ({ suggestedExcludedTags: undefined }),
+    effects: {
+      initialize: async function () {
+        this.state.suggestedExcludedTags = await getSuggestedExcludedTags()
+      },
+    },
+  }),
+  injectState,
+  ({ state, ...props }) =>
+    state.suggestedExcludedTags === undefined ||
+    props.remotes === undefined ||
+    (props.job !== undefined && props.schedules === undefined) ? (
```

Comment:
Not ideal. If we're editing a job but `suggestedExcludedTags` and `remotes` are both fetched before `job` and `schedules`, then `New` will be mounted, then unmounted then re-mounted.